components/raftstore-v2/src/operation/ready/snapshot.rs

```diff
@@ -407,13 +453,6 @@ impl<EK: KvEngine, ER: RaftEngine> Storage<EK, ER> {
             return Err(raft::Error::Store(
                 raft::StorageError::SnapshotTemporarilyUnavailable,
             ));
-        } else {
-            info!(
```

Comment:
why move this

Reply:
it can't see the snapshot state if it sending the reused snapshot.

Reply:
ok, I will readd it .